app/src/main/java/org/oppia/android/app/topic/TopicFragment.kt

```diff
@@ -34,15 +35,17 @@ class TopicFragment : InjectableFragment() {
       TOPIC_FRAGMENT_ARGUMENTS_KEY,
       TopicFragmentArguments.getDefaultInstance()
     )
-    val internalProfileId = arguments?.extractCurrentUserProfileId()?.internalId ?: -1
+    val profileId =
+      arguments?.extractCurrentUserProfileId() ?: ProfileId.newBuilder().setInternalId(-1).build()
```

Comment:
Here and elsewhere in the PR, please set the default ProfileId as fallback. We are moving away from setting  -1(logged out) as default due to reasons outlined in https://github.com/oppia/oppia-android/issues/5440.

```suggestion
      arguments?.extractCurrentUserProfileId() ?: ProfileId.getDefaultInstance()
```